sections/introduction.include

```diff
@@ -456,17 +456,18 @@
   A basic HTML document looks like this:
 
   <xmp id="intro-early-example" highlight="html">
-    <!DOCTYPE html>
-    <html>
-      <head>
-        <title>Sample page</title>
-      </head>
-      <body>
-        <h1>Sample page</h1>
-        <p>This is a <a href="demo.html">simple</a> sample.</p>
-        <!-- this is a comment -->
-      </body>
-    </html>
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <title>Document title</title>
```

Comment:
There are other sections of the prose where the original title "sample page" was referenced.  These would need to be updated.